visualization/dashboard/src/MLIDashboard/Controls/WhatIfTab/WhatIfTab.tsx

```diff
@@ -652,12 +710,7 @@ export class WhatIfTab extends React.PureComponent<IWhatIfTabProps, IWhatIfTabSt
             } else {
                 const yAxisLabels: string[] = [localization.featureImportance];
                 if (this.props.metadata.modelType !== ModelTypes.regression) {
-                    yAxisLabels.push(
-                        localization.formatString(
-                            localization.WhatIfTab.classLabel,
-                            this.props.metadata.classNames[0],
-                        ) as string,
-                    );
+                    yAxisLabels.push(this.props.weightLabels[this.props.selectedWeightVector] as string);
```

Comment:
shouldn't this be types as array of string so that the "as string" would not be needed?

Reply:
its a dictionary of strings, keyed on numbers and strings. Specifying that as the type is problematic in typescript, or at least has been